iree/base/windows.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
```

Comment:
Maybe rename to 'platform_headers' instead of windows? then we have //iree/base:platform_headers sprinkled around instead of windows specific stuff even if on other platforms. We can also move the target_platform guard inside so includers don't need to care - "just include platform_headers".

We'll want this for iOS/MacOS too, which requires a few core foundationy headers be included for anything meaningful, and android for the same logging/tracing things. I think it's fine to have them all in a single header for now.

Reply:
Good ideas, done.